packages/cdk8s-cli/lib/import/base.ts

```diff
@@ -74,13 +74,27 @@ export abstract class ImportBase {
         }
         await this.harvestPython(targetdir, moduleName);
         break;
+
+      case Language.JAVA:
+        if (moduleNamePrefix != null) {
+          // logging error instead of throwing, so it doesn't interrupt other imports
+          console.error(`Name overriding of imports is not yet supported in java. Named import: ${moduleNamePrefix}`);
+          break;
+        }
+        await this.harvestJava(targetdir, moduleName);
+        break;
   
       default:
         throw new Error(`unsupported language ${options.targetLanguage} (yet)`);
     }
   
   }
 
+  private async harvestJava(targetDir: string, moduleName: string) {
+    const target = path.join(targetDir, moduleName);
+    await fs.move(`dist/java/`, target, { overwrite: true });
```

Comment:
ok this is where I think I missed something: technically I think we should harvest only the sources (& jsii tarball) but not the pom file.

Reply:
Aha! I'm new to maven, so I was having some troubles importing `k8s` like this:

```
    <dependency>
      <groupId>k8s</groupId>
      <artifactId>k8s</artifactId>
      <version>0.0.0</version>
    </dependency>
```

without the `pom.xml`...

Did some more digging, and I can do it without the `pom.xml` using:

```
    <resources>
      <resource>
        <directory>imports/k8s/main/resources</directory>
      </resource>
      <resource>
        <directory>imports/k8s/main/java</directory>
      </resource>
    </resources>
```

Changing above code to `await fs.move(`dist/java/src/`, target, { overwrite: true });` will work (and exclude the `pom.xml`) and in fact, brings this in line with what we're doing for python as well.

I'll refactor and combine the `harvestPython()` and `harvestJava()`